test/unit/test_easy_logging.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+
+import os.path
+from pathlib import Path
+
+import pytest
+import tomlkit
+
+from snowflake.connector import EasyLoggingConfigPython
+from snowflake.connector.config_manager import CONFIG_MANAGER
+from snowflake.connector.constants import CONFIG_FILE
+
+THIS_DIR = os.path.dirname(os.path.realpath(__file__))
```

Comment:
pytest already provides a [temp file fixture](https://docs.pytest.org/en/latest/reference/reference.html#tmp-path) you should use.  In general it's not a good idea to use directories inside the source tree for temp files and directories.  See also [the tutorial](https://docs.pytest.org/en/latest/how-to/tmp_path.html#tmp-path).  This should simplify a bunch of the test setups.